libs/design/src/atoms/form/quantity-field/quantity-input/quantity-input.component.ts

```diff
@@ -44,8 +44,11 @@ export class DaffQuantityInputComponent implements OnInit, OnDestroy {
    * Instead, we listen for the change event and manually patch form control values.
    */
   _inputControl = new FormControl();
-  // TODO: should this not use this.input as the source of truth for focused?
-  focused = false;
+
+  get focused() {
```

Comment:
Type the getter function return: `get focused(): boolean {`

Reply:
updated